lib/routes/cjlu/yjsy/index.ts

```diff
@@ -0,0 +1,105 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import timezone from '@/utils/timezone';
+
+const host = 'https://yjsy.cjlu.edu.cn/';
+
+const titleMap = new Map([
+    ['yjstz', '中量大研究生院 —— 研究生通知'],
+    ['jstz', '中量大研究生院 —— 教师通知'],
+]);
+
+export const route: Route = {
+    path: '/yjsy/:cate',
+    categories: ['university'],
+    example: '/cjlu/yjstz',
```

Comment:
This example won't work since `cate` is defined as compulsory parameter

Reply:
Thanks for your reviewing! I have corrented the example of this route.